scripts/west_commands/boards.py

```diff
@@ -93,16 +95,34 @@ def do_run(self, args, _):
         for board in list_boards.find_boards(args):
             if name_re is not None and not name_re.search(board.name):
                 continue
+
+            revisions_list = 'None'
+            if board.revisions:
+                revisions_list = ''
+                for rev in board.revisions:
+                    revisions_list += rev.name + ' '
```

Comment:
Do we want the trailing space?

```suggestion
                revisions_list = ' '.join([rev.name for rev in board.revisions])
```

Reply:
Thank's, I even made it shorter with a all in one: `revisions_list = ' '.join([rev.name for rev in board.revisions]) if board.revisions else 'None'`

Reply:
Well, compliance rejected my python line for being too long, so I revert to your suggestion...